solidity/hardhat.config.ts

```diff
@@ -13,10 +13,22 @@ dotenv.config({
   example: process.env.CI ? ".env.ci.example" : ".env.example",
 })
 
+const MAINNET_PRIVATE_KEY = process.env.MAINNET_PRIVATE_KEY
+  ? [process.env.MAINNET_PRIVATE_KEY]
```

Comment:
can you also include the new environment variables in `solidity/.env.example`?

Reply:
Done in e61f0142b5066a06a634604cbb8c7d5d38625e9e. I initially skipped adding them assuming we force them to be set to non-empty values in `.env` and it is always annoying to be forced to set all those variables even if I am not working with the given chain. But I see we can have `.env` with just, for example:
```
MATSNET_PRIVATE_KEY="0x <valid key>"
MAINNET_PRIVATE_KEY=""
ETHEREUM_RPC_URL=""
ETHEREUM_PRIVATE_KEY=""
```